new_study_assistant.py

```diff
@@ -0,0 +1,624 @@
+    f.write('</div>\n')
+
+    f.write('<br>')
+
+    f.write('<div class="image" style="display:table;">')
+    f.write('<img src=\"' + file_directory + 'n_common_attribute_distribution.png\"' + 'alt=\"number of common attributes\" width=\"800\">')
+    f.write('<br>\n')
+    f.write('<div style="display:table-caption;caption-side:bottom;">Comparison between the number of common attributes in the new study and the number of common attributes in each existing study on cBioPortal.  The dashed black line indicates the number of common attributes in the new study, while the histogram shows the data for existing cBioPortal studies.</div>\n')
+    f.write('</div>\n')
+
+    f.close()
+
+#values that should be read in
+# Parse arguments
+file_directory=''
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--new_study_path", help="path to new study data")
+parser.add_argument("--study_to_drop", help="if study being tested is already on cBioPortal it may be helpful to drop that study from the analysis")
+parser.add_argument("--specific_study", help="name of specific study on the portal to test")
+parser.add_argument("--datahub_path", help="directory where the datahub is located")
+parser.add_argument("--output_pdf", action='store_true', help="flag to output pdf")
+args = parser.parse_args()
+
+new_study_path = args.new_study_path
+study_to_drop = args.study_to_drop
+specific_study = args.specific_study
+datahub_path = args.datahub_path
+output_pdf = args.output_pdf
+random_study=False
+if (new_study_path is None) and (specific_study is None):
+    random_study = True
+
+api_flag=True
+if datahub_path is not None:
+    api_flag = False
+    print "Using cBioPortal data from local datahub: " + datahub_path
+else:
+    print "cBioPortal data will be obtained via API"
+
+latex_flag=False
+if output_pdf:
+    latex_flag=True
+    print "PDF file will be output"
+
+
+#main part of the script below
+#main function
+if api_flag:
+    all_cBioPortalStudies = get_all_study_info()
+    studies, study_attributes = get_attribute_data(all_cBioPortalStudies)
+else:
+    list_of_files = find_data_files(datahub_path)
+    study_attributes, attribute_value_data = load_data_from_files(list_of_files, datahub_path)
+study_names = get_study_names(study_attributes)
+study_data_combined = study_attributes_to_df(study_attributes)
+
+
+#choose a study to serve as a "new" study
+if random_study or specific_study is not None:
+    if random_study:
+        test_study = random.choice(study_names)
+    else:
+        test_study = specific_study
+    print "Study being analyzed: " + test_study
+    study_to_drop = test_study
```

Comment:
when a study has been chosen randomly or by name, we ignore the --study_to_drop argument. Maybe add a note in the argument comment that the argument only applies when --new_study_path is used.